workspaces/vscode-base/Dockerfile

```diff
@@ -57,11 +57,9 @@ COPY --chmod=0755 --chown=root:root pl-gosu-helper.sh /usr/bin/
 
 # Make sure that code-server's entrypoint doesn't run fixuid unnecessarily
 USER root
-RUN mkdir -p /run /var/run && \
-    touch /run/fixuid.ran /var/run/fixuid.ran
+RUN mkdir -p /run /var/run \
+    && touch /run/fixuid.ran /var/run/fixuid.ran
 
 USER coder
 EXPOSE 8080
-ENTRYPOINT ["/usr/bin/pl-gosu-helper.sh", "/usr/bin/entrypoint.sh", "--auth", "none", \
-    "--disable-update-check", "--disable-telemetry", \
-    "--bind-addr", "0.0.0.0:8080", "."]
+ENTRYPOINT ["/usr/bin/pl-gosu-helper.sh", "/usr/bin/entrypoint.sh", "--auth", "none", "--disable-update-check", "--disable-telemetry", "--bind-addr", "0.0.0.0:8080", "."]
```

Comment:
Are we considering this a formatting regression?

Reply:
Hmm... I don't think it's too bad? It's annoying that making it properly multi-line would require backslashes at the end of each line.

Reply:
That's comforting to here -- that would have been a little tricky to add.

Reply:
I would note that the `RUN` lines already require backslashes to break them up anyway.